core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionPolicyManager.java

```diff
@@ -46,8 +50,16 @@
  *
  * Created by gupele on 6/29/2015.
  */
-public final class TransmissionPolicyManager implements Stoppable {
-
+public final class TransmissionPolicyManager implements Stoppable, TransmissionHandlerObserver {
```

Comment:
So we had to change the public class also, I think which is fine. However, if some of our users if they had taken dependency on this. we would break them by this change. I am not sure we might need to consider a separate release for this, mentioning that there can be some breaking changes.

Reply:
Maybe. We added four public methods to this but did not remove any; and didn't change the implementation of existing methods. That shouldn't break the contract and the code should execute as expected. Might well warrant a deeper discussion

Reply:
Hmm